src/app/app.module.ts

```diff
@@ -8,9 +8,16 @@ import { NotAuthorizedGuard } from '@app/auth/guards/not-authorized.guard';
 import { AuthorizedGuard } from '@app/auth/guards/authorized.guard';
 import { CoursesStoreService } from '@app/services/courses-store.service';
 import { CoursesService } from '@app/services/courses.service';
+import { CoursesComponent } from './features/courses/courses.component';
+import { CoursesListComponent } from './features/courses/courses-list/courses-list.component';
 
 @NgModule({
-  declarations: [AppComponent, CourseInfoComponent],
+  declarations: [
+    AppComponent,
+    CourseInfoComponent,
+    CoursesComponent,
```

Comment:
CourseInfo and Courses components needs to be declared in their separate module (and distribute this module across the application). Please follow below module structure!
![image](https://github.com/user-attachments/assets/92c6b6c1-7d29-474e-9e53-5b7e767af206)
(Obviously, it is courses.module.ts, not .scss)

Reply:
Yeah, i missed that, done!